Tests/test_zipimport.py

```diff
@@ -0,0 +1,29 @@
+#####################################################################################
+#
+#  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We should probably come up with a non-Microsoft copyright notice for new stuff...